Bambu/interface/LeptonsFiller.h

```diff
@@ -67,13 +69,16 @@ namespace mithep {
 
       TString electronsName_ = "Electrons";
       TString muonsName_ = "Muons";
+      TString electronMVAType_ = "IDEGamma2015Trig25ns";
+      //TString electronMVAType_ = "IDEGamma2015NonTrig25ns";
       TString idName_[nLeptons][32]{};
       TString verticesName_ = "PrimaryVertexes";
       TString pfCandsName_ = "PFCandidates";
       TString nopuPFCandsName_ = "PFNoPileup";
       TString puPFCandsName_ = "PFPileup";
-
+      Bool_t useMVA = kTRUE;
       Bool_t savePassing_[nLeptons][32]{};
+      ElectronIDMVA *ElectronDiscriminator = new ElectronIDMVA();
```

Comment:
Please switch ElectronDisciminator to an ElectronIDMVA object (instead of a pointer to) and remove the initialization part, or do a proper deletion of the object in the LeptonsFiller deconstructor.